### PR TITLE
[Communication] - SDK - Removing endpoint and access key EVs

### PR DESCRIPTION
**Pull request by jbeauregardb**

### CODE REVIEW

File: sdk/communication/azure-communication-phonenumbers/src/test/java/com/azure/communication/phonenumbers/PhoneNumbersIntegrationTestBase.java [hunk 1]
```diff
@@ -51,17 +47,23 @@
 
     private static final Pattern JSON_PROPERTY_VALUE_REDACTION_PATTERN =
         Pattern.compile(String.format("(?:%s)(.*?)(?:\",|\"})", JSON_PROPERTIES_TO_REDACT.toString()), Pattern.CASE_INSENSITIVE);
+    
+
```
Comment (minnieliu) [inline sdk/communication/azure-communication-phonenumbers/src/test/java/com/azure/communication/phonenumbers/PhoneNumbersIntegrationTestBase.java:51]: nit: extra lines

**Review by minnieliu — APPROVED** [review]

**Review by lsundaralingam — APPROVED** [review]
LGTM!

**Review by JianpingChen — APPROVED** [review]

**Pull request merged**